core/indexer/elasticsearch.go

```diff
@@ -261,7 +255,7 @@ func (ei *elasticIndexer) SaveRoundInfo(round int64, shardId uint32, signersInde
 
 	req := esapi.IndexRequest{
 		Index:      roundIndex,
-		DocumentID: strconv.Itoa(int(round)),
+		DocumentID: strconv.Itoa(int(roundInfo.Index)),
```

Comment:
You can use `strconv.FormatInt(roundInfo.Index, 10)`, this way we prevent errors if the index is larger than 32 bits

Reply:
Done